src/tools/binaryen-shell.cpp

```diff
@@ -77,8 +77,8 @@ static void run_asserts(size_t* i, bool* checked, Module* wasm,
   std::unique_ptr<ShellExternalInterface> interface;
   std::unique_ptr<ModuleInstance> instance;
   if (wasm) {
-    interface = make_unique<ShellExternalInterface>();
-    instance = make_unique<ModuleInstance>(*wasm, interface.get());
+    interface = wasm::make_unique<ShellExternalInterface>();
```

Comment:
> Does VS2015 default to C++14?

Yes, and `make_unique` is present since VS2013 (June 28, 2013 https://blogs.msdn.microsoft.com/vcblog/2013/06/28/c1114-stl-features-fixes-and-breaking-changes-in-vs-2013/)

I can send a PR with AppVeyor to verify the changes against VS2015 Update 3 if this is acceptable.